### ISSUE TITLE
CustomMark style for ORFrame broken in 1.7.6

### ISSUE THREAD

**Issue by bengarvey** [emeeks/semiotic#147]
Follow up to this:
https://github.com/emeeks/semiotic/issues/76

Issue #76 was resolved, but I have an issue with this code:

```
      <ResponsiveORFrame
        { ...sharedProps }
        projection={'horizontal'}
        size={[300,700]}
        responsiveWidth={true}
        type={{
          type: "bar",
          customMark: (d, i, xy) => [
            <rect
              style={{ fill: d.funnelKey, fillOpacity: d.fillOpacity }}
              x={0}
              width={xy.width}
              height={xy.height}
            />, 
            <image
              href={d.stepValue > 0 ? `./images/${d.image}.png` : ''} 
              height="30px"
              y={0}
              x={xy.width-30}
            />  
          ]   
        }}  
        defined={d => d.total > 10} 
        data={display}
        margin={{top: 5, bottom: 25, left: 5, right: 5}} 
        oAccessor={"stepName"}
        rAccessor={"stepValue"}
        oPadding={5}
        renderMode={d => "normal"}
        oLabel={ d => <text y={5} x={9} className="barText">{d}</text> }
        style={d => {return { fill: d.funnelKey, fillOpacity: d.fillOpacity}}}
        axis={axis}
      />  
    </div>
```

The style in the customMark is not being applied and neither is the image. It's supposed to look like this:
![screen shot 2018-02-15 at 10 54 10 pm](https://user-images.githubusercontent.com/1342331/36292922-3a3393aa-12a3-11e8-93af-aa752e7bd69a.png)

But instead it looks like this:
![screen shot 2018-02-15 at 10 54 28 pm](https://user-images.githubusercontent.com/1342331/36292925-44bc8700-12a3-11e8-97b5-8cc9a932e490.png)

**Comment by emeeks** [comment]
Will take a look and try to get a fix out tomorrow.

**Comment by emeeks** [comment]
This should be fixed in 1.7.11. Reopen if you still see problems.

**Comment by bengarvey** [comment]
![screen shot 2018-02-17 at 10 22 24 am](https://user-images.githubusercontent.com/1342331/36342467-29095cf6-13cd-11e8-8326-bb0a7c2a63f7.png)
Looks good! Thanks.
